habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -143,13 +145,31 @@ def _try_acquire_context(self):
         if self.habitat_config.concur_render:
             self.renderer.acquire_gl_context()
 
+    def _auto_sleep(self):
+        all_robo_pos = [
+            robot.base_pos for robot in self.robots_mgr.robots_iter
+        ]
+        rom = self.get_rigid_object_manager()
+        for handle, ro in rom.get_objects_by_handle_substring().items():
+            is_far = all(
+                (robo_pos - ro.translation).length()
+                > self.habitat_config.sleep_dist
```

Comment:
Accessing hydra config. We will need to change similar to the other [PR](https://github.com/facebookresearch/habitat-lab/pull/1137/). I will remember to update this.

Reply:
Thanks Sriram!